orchestra/todos/import_export.py

```diff
@@ -16,6 +16,7 @@
 REMOVE_IF_HEADER = 'Remove if'
```

Comment:
In unit tests, it's worth adding a required to-do and ensuring that
* all to-dos other than it are not required
* it is required?